SD-K9/src/index.html

```diff
@@ -16,6 +16,8 @@
   <!-- add to homescreen for ios -->
   <meta name="apple-mobile-web-app-capable" content="yes" />
   <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+  <!-- remove comment tag when able to get api key from environment variables
+  <script src="https://maps.googleapis.com/maps/api/js?libraries=places&key=PUT_THE_API_KEY_HERE"></script> -->
```

Comment:
Can we remove this if it is not necessary?

Reply:
Fixed, ty.